Answers.md

```diff
@@ -0,0 +1,14 @@
+1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+Semantic html simply refers to any HTML tag which is descriptive. For instance, anything wrapped in a <section> would obviously be described as a section within the code structure. Anything within an <address> tag would be address information. This is not only for developers to easily comprehend the code but also for SEO purposes. When one searches Google for an address, these search engines can easily extract the data they need, like from the <address> tag, and show you the results you're looking for.
+
+2. Describe some differences between display: block; and display: inline;.
+Display block means the element will take up all the width. Subsequent elements will not be displayed on the same horizonal axis. Display inline means subsequent elements will appear on the horizontal axis, side-by-side to other elements.
+
+3. What are the 4 areas of the box model?
+Margin, border, padding, content
+
+4. While using flexbox, what axis are you using when you use the property: align-items: center?
+The off-axis
```

Comment:
Close! The correct answer is **cross-axis**. 

Here is a great resource for understanding that: https://css-tricks.com/snippets/css/a-guide-to-flexbox/